src/classes/BDI_AdditionalObjectService_TEST.cls

```diff
@@ -1305,9 +1305,9 @@ private class BDI_AdditionalObjectService_TEST {
                         Data_Import_Object_Mapping_Set__c = predecessorObjMapping.Data_Import_Object_Mapping_Set,
                         Object_API_Name__c = 'Contact',
                         Predecessor__c = predecessorObjMapping.DeveloperName,
-                        Imported_Record_Field_Name__c = 'Contact1_Title__c',
-                        Imported_Record_Status_Field_Name__c = 'Contact1ImportStatus__c',
-                        Relationship_Field__c = 'Honoree_Contact__c',
+                        Imported_Record_Field_Name__c = UTIL_Namespace.StrTokenNSPrefix('Contact1_Title__c'),
+                        Imported_Record_Status_Field_Name__c = UTIL_Namespace.StrTokenNSPrefix('Contact1ImportStatus__c'),
```

Comment:
Style update only, but can we avoid the long line and add a linebreak in here?

Reply:
I don't think we can do a line break here. It doesn't look excessively long in VSCode.